central/graphql/resolvers/compliance.go

```diff
@@ -617,11 +616,8 @@ func (resolver *complianceControlResolver) ComplianceControlEntities(ctx context
 	if err != nil || !hasComplianceSuccessfullyRun {
 		return nil, err
 	}
-	store, err := resolver.root.NodeGlobalDataStore.GetClusterNodeStore(ctx, clusterID, false)
-	if err != nil {
-		return nil, err
-	}
-	return resolver.root.wrapNodes(store.ListNodes())
+
+	return resolver.root.wrapNodes(resolver.root.NodeDataStore.SearchRawNodes(ctx, clusterQuery(clusterID)))
```

Comment:
does this return more now than before? iirc, `List` something returns parts of the full proto vs the whole proto, right?

Reply:
The `ListNode()` always pulled full nodes, so to answer your questions the #bytes pulled is the same as before.